tests/cli-v2.rs

```diff
@@ -804,7 +804,7 @@ fn add_target_v1_toolchain() {
                 clitools::CROSS_ARCH1,
                 "--toolchain=nightly",
             ],
-            for_host!("toolchain 'nightly-{0}' does not support components"),
+            for_host!("Missing manifest in toolchain 'nightly-{0}'"),
```

Comment:
This error message change seems bogus - There is a manifest, it's just a V1 manifest which doesn't support the component concept, no?

Reply:
hmm, lets see https://github.com/rust-lang/rustup/pull/2721/files/8add44771eccee37e53964dcd9fa8d8f2a3d812c#diff-1e51556c6aa8a6243798d60243781b6acabd184d6d26c6c5f484eefd7b9e49c2R923 is the key bit: so this is (load it up fully) - 
```
remove_component()
 if let Some(manifest) = manifestation.load_manifest()? {
   ...} else {
     -> change here
     Err MissingManifest
   -> .change finishes
 ... }

I don't recall what led to this delta. But its clearly a manifest that doesn't load from manifestation.load_manifest(), which doesn't know how to load V1 manifests.

I don't think this code is wrong with this change, but I agree its not related to anyhow. Want it backed out?

Reply:
I'm not terribly worried since V1 manifests are only for the very oldest toolchains anyway.  let's not back it out for now.